### DISCUSSION TITLE
Derived column with regex `*`

### DISCUSSION THREAD

**Discussion by piec** [saulpw/visidata#2448] [Q&A] [answered]
Hello,

I'm trying to use 
```
      * regex/subst   add column derived from current column, replacing regex with subst (may include \1 backrefs)
```
(See also https://www.visidata.org/docs/columns/#player-split-regex)

But I can't get it to work.
In the following video: [link](https://github.com/saulpw/visidata/assets/781471/35de877e-083b-458c-b601-465c847f5d22) I'm using `*` then `Food/a` to create a new column that has "Food" replaced by "a", but the new column values are not changed, am I doing something wrong?

Thanks,
Pierre

**Comment by midichef** [comment, marked as answer]
Hi, thanks for including the video and using one of the sample data files in visidata, that's very helpful.

When using `*` there are two separate fields that you can fill out, `search` and `replace`.  Pressing `TAB` will move between them. So the key sequence would be:  `*` `Food` `Tab` `a` `Enter`

**Reply by piec** [reply]
Oh right, I thought `replace:` was some kind of title
Thanks :)
